src/Extension/TranslationTrait.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Bolt\Extension;
+
+use Bolt\Filesystem\Handler\DirectoryInterface;
+use Pimple as Container;
+
+/**
+ * Twig function/filter addition and interface functions for an extension.
+ *
+ * @author Carson Full <carsonfull@gmail.com>
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+trait TranslationTrait
+{
+
+    /**
+     * Call this in register method.
+     *
+     * @internal
+     */
+    final protected function addTranslations()
```

Comment:
In line with the other functions, the should be named something more like `extendTranslatorService`